src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -17,17 +27,34 @@ export type Props = {
 };
 
 export type State = {
+    activeMention: Mention | null;
+    containerRect: DOMRect | null;
     editorState: EditorState;
+    mentionPattern: RegExp;
+    popupReference: VirtualElement | null;
 };
 
 export default class ReplyField extends React.Component<Props, State> {
+    static defaultProps = {
+        className: '',
+        isDisabled: false,
+        mentionTriggers: ['@', '＠', '﹫'],
```

Comment:
Do these work for all locales?

Reply:
> Do these work for all locales?

Copied from BUIE. I think it'll work fine since BUIE used them for a long time.